### DISCUSSION TITLE
Files are uploaded successfully, but the model doesn't detect the file.

### DISCUSSION THREAD

**Discussion by timmanik** [danny-avila/LibreChat#3980] [Troubleshooting] [answered]
Hi,

I am able to upload files. The **rag_api** service is running.

However, when after uploading the file into the chatbox, the model doesn't seem to detect that there is a file.

Attached is a screenshot of the docker-compose logs. As you can see, there is a successful POST request made to the rag_api service.

<img width="1096" alt="image" src="https://github.com/user-attachments/assets/ca05892f-dda3-4b08-8344-ce562257245c">

Here is a screenshot of the chatbox.

<img width="1259" alt="image" src="https://github.com/user-attachments/assets/644ee1a5-63d5-42d0-9486-99d72fa217c3">

I am using huggingface as my embeddings provider. Here is how I defined it:

<img width="462" alt="image" src="https://github.com/user-attachments/assets/25afe56c-6db6-4b6b-81f0-80e6fde0ac15">

Below is my docker-compose.override.yml

<img width="452" alt="image" src="https://github.com/user-attachments/assets/5fb3a098-ecfc-42fc-bb9e-70139556b8e1">

Happy to provide additional info as well.

**Comment by timmanik** [comment]
Update: This _may_ be a bug with the new native Bedrock integration. I tried file uploads with other endpoints like Anthropic and Ollama. Those endpoints are able to detect the file and chat about the file.

**Reply by danny-avila** [reply, marked as answer]
Not implemented yet for Bedrock, will be within the week.

**Reply by timmanik** [reply]
Thanks, Danny. Great work on the native AWS Bedrock launch.

**Reply by ldavis9000aws** [reply]
@timmanik Ran into this today as well. I'm using LiteLLM for Bedrock as a workaround for any attachments I want to add until the issue is resolved.

**Comment by danny-avila** [comment]
This should now be resolved with this PR: https://github.com/danny-avila/LibreChat/pull/4322
